.github/chainguard/cross-distro.sts.yaml

```diff
@@ -0,0 +1,10 @@
+issuer: https://accounts.google.com
+
+# TODO: get subject_pattern
+subject_pattern: "TODO"
```

Comment:
While probably fine, let's hold off on submitting this until we have the subject. (also we should try to use `subject` over `subject_pattern` if we can)